js/src/ux.js

```diff
@@ -4,11 +4,13 @@ import Ui from "./Ui.js";
 import App from "./App.js";
 
 import tools from "./tools/tools.js";
+import { Locale } from "./locale.js";
 
 const ux = {
     Ui,
     App,
-    tools
+    tools,
+    locale: Locale,
```

Comment:
What is the purpose of changing name to lowercase here?

Reply:
Following camelCase variable naming convention.

Reply:
Then why are you exporting/importing it as `Locale`? It's inconsistent.

Reply:
If it's a class, then i believe it should be named `Locale`, if it's not - it should be `locale`, in all places.

Reply:
I disagree. Keep in mind that ux.locale is a variable which can have anything assigned to it. Example scenario would be having 2 different localization libraries e.g. `Locale` and `DifferentLocale` that implement same API. Now, you could seemlessly switch between the two, because the `ux` API wouldn't change, but if you named variable same as class import, then you would break compatibility by making the `ux.DifferentLocale` switch.

Reply:
I see. Ok, let's just leave it as it is now.